src/components/Article.jsx

```diff
@@ -1,27 +1,53 @@
-import { useEffect, useState } from "react";
-import getArticles from "../../api";
-
-export default function Articles() {
-    const [articles, setArticles] = useState([]);
+import { useParams } from "react-router-dom";
+import { getArticlesById } from "../../api";
+import { useState } from "react";
+import { useEffect } from "react";
+import { Link } from "react-router-dom";
 
+export default function Article() {
+    const {id} = useParams();
+    const [article, setArticle] = useState(null);
+    const [isLoading, setIsLoading] = useState(true);
+    
     useEffect(() => {
-        getArticles()
-        .then((articlesData) => {
-            setArticles(articlesData.sortedArticles);
+        getArticlesById(id)
+        .then((articleData) => {
+            setArticle(articleData.article);
+            setIsLoading(false);
+        })
+        .catch((error) => {
+            return error;
         })
     }, [])
 
-    return (
-        <>
-        {articles.map((article) => {
-            console.log(article.article_img_url)
-            return (
-                <div className="article-card">
-                    <h1>{article.title}</h1>
-                    <p>{article.author}</p>
-                </div>
-            )
-        })}
-        </>
-    )
-} 
\ No newline at end of file
+    // if (error) {
+    //     return <h1>{error}</h1>;
+    // }
+
+    return isLoading ? (
+        <h1>Loading...</h1>
+      ) : (
+        <div>
+            <h1 className="article-title">{article.title}</h1>
+            <Link to={``}>
+                <h2>{article.author}</h2>
+            </Link>
+            <h2>{article.created_at}</h2>
+            <br></br>
+            <br></br>
+            <img className="article-image" src={article.article_img_url} alt="Article Image"></img>
```

Comment:
<img className="article-image" src={article.article_img_url} alt="Article Image" />
      
      you can close an image tag like this